CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,77 @@
+- Add Support for `FlexibleIPAM` on Antrea Multicast so that multicast packets can be forwarded to the uplink interface when `FlexibleIPAM` is enabled. ([#4922](https://github.com/antrea-io/antrea/pull/4922), [@ceclinux])
+- Support Talos clusters to run Antrea as the CNI, and add Talos to the K8s installers document. ([#5718](https://github.com/antrea-io/antrea/pull/5718) [#5766](https://github.com/antrea-io/antrea/pull/5766), [@antoninbas])
+
+### Changed
+
+- Support secondary network when the network configuration in `NetworkAttachmentDefinition` does not include IPAM configuration. ([#5762](https://github.com/antrea-io/antrea/pull/5762), [@jianjuns])
+- Change secondary network Pod controller to subscribe to CNIServer events to support bridging and VLAN network. ([#5767](https://github.com/antrea-io/antrea/pull/5767), [@jianjuns])
+- Use Antrea IPAM for Antrea native secondary network support. ([#5427](https://github.com/antrea-io/antrea/pull/5427), [@jianjuns])
+- Create different images for antrea-agent and antrea-controller to minimize the overall image size, speeding up the startup of both antrea-agent and antrea-controller. ([#5856](https://github.com/antrea-io/antrea/pull/5856) [#5902](https://github.com/antrea-io/antrea/pull/5902) [#5903](https://github.com/antrea-io/antrea/pull/5903), [@jainpulkit22])
+- Delete OVS port and flows before releasing Pod IP. ([#5788](https://github.com/antrea-io/antrea/pull/5788), [@tnqn])
+- Don't create tunnel interface (antrea-tun0) when using Wireguard encryption mode. ([#5885](https://github.com/antrea-io/antrea/pull/5885) [#5909](https://github.com/antrea-io/antrea/pull/5909), [@antoninbas])
+- Update Windows documentation with more clearer installation guide and instructions. ([#5789](https://github.com/antrea-io/antrea/pull/5789), [@antoninbas])
+- Enable IPv4/IPv6 forwarding on demand automatically to eliminate the need for user intervention or dependencies on other components. ([#5833](https://github.com/antrea-io/antrea/pull/5833), [@tnqn])
+- Prioritize L7 NetworkPolicy flows over `TrafficControl` to avoid a potential issue that a `TrafficControl` CR with a redirect action to the same Pod could bypass the L7 engine. ([#5768](https://github.com/antrea-io/antrea/pull/5768), [@hongliangl])
+- Add ability to skip loading kernel modules in antrea-agent to support some specialized distributions (eg: Talos). ([#5754](https://github.com/antrea-io/antrea/pull/5754), [@antoninbas])
+- Add NetworkPolicy rule name in Traceflow observation. ([#5667](https://github.com/antrea-io/antrea/pull/5667), [@Atish-iaf])
+- Use Traceflow API v1beta1 to replace the deprecated API version in `antctl traceflow`. ([#5689](https://github.com/antrea-io/antrea/pull/5689), [@Atish-iaf])
+- Replace `net.IP` with `netip.Addr` in FlowExporter. ([#5532](https://github.com/antrea-io/antrea/pull/5532), [@antoninbas])
```

Comment:
It should mention the optimiation of memory usage, otherwise the change looks meaningless to users

Reply:
Done.